lib/cocoapods/target/aggregate_target.rb

```diff
@@ -7,6 +7,9 @@ class AggregateTarget < Target
     #         generated this target.
     attr_reader :target_definition
 
+    # Product types where the product's frameworks must be embedded in a host target
+    EMBED_FRAMEWORKS_IN_HOST_TARGET_TYPES = [:app_extension, :watch_extension].freeze
```

Comment:
Maybe my naming is bad, but I don't think I can add frameworks to this list. Adding a type to this list means that the analyzer will go looking for the target's host in your project. I don't have a clear picture in my head of what the framework-type situation would look like, but if we're talking about projects meant for building/developing a single framework/lib, then I don't think it'll find a host target.